owners/src/parser.js

```diff
@@ -37,23 +46,29 @@ class OwnersParser {
    * Parse an OWNERS.yaml file.
    *
    * @param {!string} ownersPath OWNERS.yaml file path.
-   * @return {OwnersRule} parsed OWNERS file rule.
+   * @return {OwnersParserResult} parsed OWNERS file rule.
    */
   parseOwnersFile(ownersPath) {
     const contents = this.localRepo.readFile(ownersPath);
     const lines = yaml.parse(contents);
+    const errors = [];
+    const rules = [];
 
-    if (!(lines instanceof Array)) {
-      this.logger.warn(
-        `Failed to parse file '${ownersPath}'; must be a YAML list`
+    if (lines instanceof Array) {
+      const stringLines = lines.filter(line => typeof line === 'string');
+      const ownersList = stringLines.filter(line => line.indexOf('/') === -1);
```

Comment:
couldn't you do this in one filter? doesn't look like you need the intermediate stringLines var

Reply:
This sets up for later PRs which will be splitting rules into dicts and strings, since dicts will be used for pattern-based rules.